crew

```diff
@@ -439,14 +457,14 @@ def install_package (pkgdir)
     FileUtils.mv 'filelist', CREW_CONFIG_PATH + "meta/#{@pkg.name}.filelist"
 
     # Strip libraries with -S
-    system "find . -name 'lib*.a' -print | xargs chmod u+w 2>/dev/null" unless CREW_NOT_STRIP
-    system "find . -name 'lib*.a' -print | xargs strip -S 2>/dev/null" unless CREW_NOT_STRIP
-    system "find . -name 'lib*.so*' -print | xargs chmod u+w 2>/dev/null" unless CREW_NOT_STRIP
-    system "find . -name 'lib*.so*' -print | xargs strip -S 2>/dev/null" unless CREW_NOT_STRIP
+    system "find . -type f -name 'lib*.a' -print | xargs -r chmod u+w" unless CREW_NOT_STRIP
+    system "find . -type f -name 'lib*.a' -print | xargs -r sh -c 'for i in \"$0\" \"$@\"; do case \"$(head -c 4 $i)\" in ?ELF|\!?ar) echo \"$i\";; esac ; done' | xargs -r strip -S" unless CREW_NOT_STRIP
+    system "find . -type f -name 'lib*.so*' -print | xargs -r chmod u+w" unless CREW_NOT_STRIP
+    system "find . -type f -name 'lib*.so*' -print | xargs -r sh -c 'for i in \"$0\" \"$@\"; do case \"$(head -c 4 $i)\" in ?ELF) echo \"$i\";; esac ; done' | xargs -r strip -S" unless CREW_NOT_STRIP
 
     # Strip binaries
-    system "find . -type f -perm /111 -print | sed -e '/lib.*\.a$/d' -e '/lib.*\.so/d' | xargs chmod u+w 2>/dev/null" unless CREW_NOT_STRIP
-    system "find . -type f -perm /111 -print | sed -e '/lib.*\.a$/d' -e '/lib.*\.so/d' | xargs strip 2>/dev/null" unless CREW_NOT_STRIP
+    system "find . -type f -perm /111 -print | sed -e '/lib.*\.a$/d' -e '/lib.*\.so/d' | xargs -r chmod u+w" unless CREW_NOT_STRIP
+    system "find . -type f -perm /111 -print | sed -e '/lib.*\.a$/d' -e '/lib.*\.so/d' | xargs -r sh -c 'for i in \"$0\" \"$@\"; do case \"$(head -c 4 $i)\" in ?ELF) echo \"$i\";; esac ; done' | xargs -r strip" unless CREW_NOT_STRIP
 
```

Comment:
Is it a safe assumption to assume _ALL_ shared library object files start with `^lib`?  Wouldn't we potentially miss other library files that need stripped?